daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/PartialNextElementResolver.scala

```diff
@@ -76,12 +76,8 @@ trait NextElementResolver { self: InfosetInputter =>
         // set of possible elements.
         //
         maybeERD = resolver.maybeNextElement(name, nameSpace, hasNamespace)
-        //
-        // If the ERD is marked hidden, then we never expect any events corresponding
-        // to its elements.
-        //
-        if (maybeERD.isDefined && maybeERD.get.isHidden)
-          maybeERD = Nope // clobber it back to Nope
+        // we address the cases where ERD is in a hidden context where the elements are assigned
```

Comment:
Rephrase. "The cases where .... are addressed where..." 
I had a hard time following the comment in the current wording.